### PR TITLE
fuel prices & exchange rates

### PR DESCRIPTION
**Pull request by heychirag**
The latest currency exchange rates from [fixer.io](http://fixer.io/) and fuel prices from [globalpetrolprices.com](http://www.globalpetrolprices.com/api/index.php) will be downloaded and saved to browser storage. Also an attempt will be made to update this data on weekly basis.

Note:

- The api I have used is free **but** provides exchange rates for 31 most popular currencies only. The [yahoo api](http://finance.yahoo.com/) you suggested isn't well documented and is likely to be constricted with api calls quota.

- I have not received any confirmation from neven.valev@globalpetrolprices.com. If the API stops working in future, we might have to contact him again.

### CODE REVIEW

File: Source/manifest.json [hunk 1]
```diff
@@ -259,6 +259,7 @@
     "debugger",
     "idle",
     "geolocation",
-    "*://en.wikipedia.org/*"
+    "*://en.wikipedia.org/*",
+     "*://www.globalpetrolprices.com/*"
```
Comment (Ceilican) [inline Source/manifest.json]: wrong indentation in this line.

**Comment by Ceilican** [comment]
Thanks!

**Comment by Ceilican** [comment]
Are you ensuring that the app has and uses default fuel prices and exchange rates, in case communication with fixer.io and globalpetrolprices.com fails?

**Comment by heychirag** [comment]
In #75, I have added this functionality as well

**Pull request merged**